src/credential/sd_jwt.rs

```diff
@@ -0,0 +1,341 @@
+use super::{Credential, CredentialFormat, ParsedCredential, ParsedCredentialInner};
+use crate::{oid4vp::permission_request::RequestedField, CredentialType, KeyAlias};
+
+use std::sync::Arc;
+
+use openid4vp::core::presentation_definition::PresentationDefinition;
+use ssi::{
+    claims::{
+        sd_jwt::SdJwtBuf,
+        vc::v2::{Credential as _, JsonCredential},
+        vc_jose_cose::SdJwtVc,
+    },
+    prelude::AnyJsonCredential,
+};
+use uniffi::deps::log;
+use uuid::Uuid;
+
+#[derive(Debug, uniffi::Object)]
+pub struct SdJwt {
+    pub(crate) id: Uuid,
+    pub(crate) key_alias: Option<KeyAlias>,
+    pub(crate) credential: JsonCredential,
+    pub(crate) inner: SdJwtBuf,
+}
+
+// Internal utility methods for decoding a SdJwt.
+impl SdJwt {
```

Comment:
Would much prefer to call this something like `VcdmSdJwt`, as we may want to support non-VCDM SD-JWTs in the future.

Reply:
Opened https://github.com/spruceid/mobile-sdk-rs/pull/63 to track.